syncmap_test.go

```diff
@@ -0,0 +1,132 @@
+package dapper_test
+
+import (
+	"sync"
+	"testing"
+)
+
+type syncmaps struct {
+	Map sync.Map
+}
+
+// This test verifies that that sync.Map key/value types are rendered when
+// they can be inferred from the context.
+func TestPrinter_SyncMap(t *testing.T) {
+	var m sync.Map
+
+	test(t, "empty sync.SyncMap", &m, "*sync.Map{}")
+
+	m.Store(1, 100)
+	m.Store(2, 200)
+
+	test(
+		t,
+		"*sync.SyncMap",
+		&m,
+		"*sync.SyncMap{",
+		"    int(1): int(100)",
+		"    int(2): int(200)",
+		"}",
+	)
+}
+
+// This test verifies the formatting of sync.Map key/values in the named structs.
+func TestPrinter_SyncMapInNamedStruct(t *testing.T) {
+	test(
+		t,
+		"empty sync.Map in named struct",
```

Comment:
```suggestion
		"empty sync.Map",
```

These descriptions don't have to repeat the test name (the function itself), it's already included in the test output.